drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -14,6 +14,10 @@ config WIFI_NRF700X
 	help
 	  Nordic Wi-Fi Driver
 
+config NRF_WIFI_LOW_POWER
+	bool "Enable low power mode in nRF Wi-Fi chipsets"
+	default n
```

Comment:
Any reason not to set this to `y` by default?

Reply:
Just couple of days back got approval from QA that this is working, so, have a local commit which enabled this by default, will update this branch too.